website/docs/tutorial/using-jinja.md

```diff
@@ -5,7 +5,7 @@ id: "using-jinja"
 
 In this tutorial, we're going to take a common pattern used in SQL, and then use Jinja to improve our code.
 
-If you'd like to work through this query, add [this CSV](https://github.com/dbt-labs/jaffle_shop/blob/master/data/raw_payments.csv) to the `data/` folder of your dbt project, and then execute `dbt seed`.
+If you'd like to work through this query, add [this CSV](https://github.com/dbt-labs/jaffle_shop/blob/HEAD/seeds/raw_payments.csv) to the `seeds/` folder of your dbt project, and then execute `dbt seed`.
```

Comment:
this will be a broken link until we update the `jaffle_shop` repo - what do you think of leaving this as is for now, and leaving ourselves a `TODO`?

Reply:
that works!